### PR TITLE
Add React Prep Work

### LINKED ISSUE #66: Add Pre-Module Work to the React Module
**Which module(s) and week(s) does this change effect?**
Module(s): React
Week(s): Pre-Module

**What is the work that needs to be done?**

There is some pre-work that has to be done before the React module starts.

You can find a discussion about that [here](https://codeyourfuture.slack.com/archives/C76GQDRH9/p1600431497001100)

This should be added to both the `Overview` and `Lesson 1` pages of the React module so teachers do not miss it. You could consider adding a `Prep` page to the syllabus so it can be directly linked to.

**Why is this work important to do?**

If we leave this work to be done in the lesson we lose a precious lesson time to simple work that can be done at home 

**Who might need to know about this change?**

@40thieves @nbogie

### PR DESCRIPTION
**Pull request by ChrisOwen101**
See issue for details

Closes #66

### CODE REVIEW

File: docs/react/prep.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+id: prep
+title: React Preparation
+sidebar_label: Preparation
+---
+
+Before you start React you must make sure that you have some additional software is installed.
```
Comment (illicitonion) [inline docs/react/prep.md]:
```suggestion
Before you start React you must make sure that you have some additional software installed.
```

"you have", or "is", but not both!

File: docs/react/prep.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+You should already have these tools installed.
+
+To check if Node is installed open `Command Prompt` or `Terminal` or `Git Bash` and type.
+
+```
+node -v
+```
+
+If this does not work, you should [install Node](https://nodejs.org/en/download/).
```
Comment (illicitonion) [inline docs/react/prep.md]:
```suggestion
Running this command should print out a version number. If this does not work, you should [install Node](https://nodejs.org/en/download/).
```

It's not always obvious what working looks like :)

File: docs/react/week-1/lesson.md [hunk 3]
```diff
@@ -10,6 +10,14 @@ Click [here](./learning-objectives) to see the Learning Objectives for this less
 
 ---
 
+## Before you start
+
+Before you arrive at class you **must** have completed the following setup work.
+
+[React Preparation](../prep)
+
+_If you are a teacher reading this, please check the [Instructor Notes](./mentors)_
```
Comment (illicitonion) [inline docs/react/week-1/lesson.md]:
```suggestion
_If you are a teacher reading this, please check the [Instructor Notes](./instructors)_
```

mentors.md re-identifies itself with an `id: instructors` directive

File: docs/react/week-1/mentors.md [hunk 4]
```diff
@@ -4,6 +4,14 @@ title: Instructor Notes
 sidebar_label: Instructor Notes
 ---
 
+## IMPORTANT - Setup Needed For Students
+
+If you are teaching this class in the coming weeks you **must** instruct your students to complete the instructions below.
+
+[React Preparation](./prep)
```
Comment (illicitonion) [inline docs/react/week-1/mentors.md]:
```suggestion
[React Preparation](../prep)
```

`prep` is in the `/react` folder not the `/react/week-1` folder

File: docs/react/week-1/mentors.md [hunk 5]
```diff
@@ -4,6 +4,14 @@ title: Instructor Notes
...
+
+If you students do not arrive with these steps completed then you will lose a lot of time on your first lesson. You should consider running short mid-week sessions the week before to help students get setup.
```
Comment (illicitonion) [inline docs/react/week-1/mentors.md]:
```suggestion
If you students do not arrive with these steps completed then you will lose a lot of time on your first lesson. You should consider running short mid-week sessions the week before to help students get set up.
```

**Review by illicitonion — APPROVED** [review]
Looks good :) I've left a bunch of minor suggestions about the markup, but the content looks great

**Comment by 40thieves** [comment]
Late to the party, but this looks good to me, thanks for taking it on @ChrisOwen101!

**Pull request merged**